.circleci/config.yml

```diff
@@ -243,8 +244,7 @@ jobs:
       - store_test_results:
           path: /go/out/tests
 
-
-  e2e-pilot-auth:
+  e2e-pilot-alpha3:
```

Comment:
`e2e-pilot-alpha3-noauth` instead?

Reply:
Since it executes `make test/minikube/noauth/e2e_pilot HUB="${HUB}" TAG="${TAG}"`

Reply:
Will rename them in separate PR.